terraform/projects/infra-database-backups-bucket/bucket_policy.tf

```
@@ -18,9 +18,7 @@ data "aws_iam_policy_document" "database_backups_cross_account_access" {
     principals = {
       type = "AWS"
 
-      identifiers = [
-        "${var.database_backups_access_list}",
-      ]
+      identifiers = "${var.database_backups_access_list}"
```

Shouldn't it just be this?

```suggestion
      identifiers = var.database_backups_access_list
```

(Might not be possible in the current version of Terraform, but worth a `plan` to see. Otherwise it looks like we're setting a string, not an array)

---

Due to the formatting of the version, the change gives a linting error